uk.co.compucorp.civicrm.hrcore/backstop_data/scenarios/contact-actions-menu.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We already had a scenario for the action menu, `"Contact Summary / Actions"` under `contact-summary.json`.  So I think we are duplicating the tests. So,
1. Can we not use the old one as the button to open the menu seems to be the same? if there is some modifications needed then we should change the old one. And also add `Contact Action Menu - User deletetion confirmation dialog` to the `contact-summary.json` itself.

2. IF for some reason you think we cant use the previous json file, please make sure to delete ``"Contact Summary / Actions"`` and leave reason in the doc.

Also please make sure that none of the old backstop tests are producing errors as regression.

Reply:
Also please place any action menu related casper scripts in the `contact-summary` folder as there we previously had the related js files.